build.gradle

```diff
@@ -191,6 +191,12 @@ subprojects {
     //Required for building on Travis' container-based infrastructure to not be killed with
     //'exit code 137' - https://github.com/travis-ci/travis-ci/issues/5582
     jvmArgs '-Xmx512m'
+    if (rootProject.ext.javaVersion >= 17) {
```

Comment:
Probably 16, but it might be unlikely that anyone builds with JDK 16 anymore.

Reply:
This is only for own internal Test Gradle-Task executions and we do not execute with Java 16.

Reply:
That is sacred knowledge, but really too minor to argue about it. Totally fine with me. 😉